pkg/agent/agentversionsjob.go

```diff
@@ -0,0 +1,201 @@
+package agent
+
+import (
+	"bytes"
+	"encoding/xml"
+
+	"fmt"
+	"io/ioutil"
+	"regexp"
+	"strconv"
+	"strings"
+
+	"net/http"
+
+	"github.com/Axway/agent-sdk/pkg/jobs"
+	"github.com/Axway/agent-sdk/pkg/util/errors"
+	log "github.com/Axway/agent-sdk/pkg/util/log"
+)
+
+var agentURL = map[string]string{
+	"AWSDiscoveryAgent":                      "aws-apigw-discovery-agent",
+	"AWSTraceabilityAgent":                   "aws-apigw-traceability-agent",
+	"AzureDiscoveryAgent":                    "azure-discovery-agent",
+	"AzureTraceabilityAgent":                 "azure-traceability-agent",
+	"EnterpriseEdgeGatewayDiscoveryAgent":    "v7-discovery-agent",
+	"EnterpriseEdgeGatewayTraceabilityAgent": "v7-traceability-agent",
```

Comment:
Should the istio agent be here too?

Reply:
The other two options on jfrog are config-sync-agent, and service-mesh-agent. I can map those two build names to those agents if available.

Reply:
Those are meshv1 agents. No options for the meshv2 agents? als-traceability-agent or discovery-agent

Reply:
I chatted with Trevor and opted to add those two in another story, as the location of their docker releases are structured differently. Will add those after they are aligned with these agents for continuity. 
https://jira.axway.com/browse/APIGOV-19578

Reply:
Could this be configured by the agent, so we don't have to keep this mapping

Reply:
@vivekschauhan That was the original plan, to send in this info from the agents. However, if this solution does change eventually, I chose to keep all of the related code in one place. I am open to removing the mapping, or making a change part of an APIGOV-19578 solution.